.ci/scripts/release/github-release.sh

```diff
@@ -4,7 +4,8 @@ export GITHUB_TOKEN=${GITHUB_TOKEN_PSW}
 export GITHUB_ORG=zeebe-io
 export GITHUB_REPO=zeebe
 
-curl -sL https://github.com/aktau/github-release/releases/download/v0.7.2/linux-amd64-github-release.tar.bz2 | tar xjvf - --strip 3
+curl -sL https://github.com/meterup/github-release/releases/download/v0.7.5/linux-amd64-github-release.bz2 | bzip2 -fd - > github-release
```

Comment:
bzip2 is always writing the decompressed content to the output, thus the  `bzip2 -fd - > github-release` command